src/actions/channels.ts

```diff
@@ -1445,6 +1445,32 @@ export function membersMinusGroupMembers(channelID: string, groupIDs: Array<stri
     });
 }
 
+export function getChannelModerations(channelId: string): ActionFunc {
+    return bindClientFunc({
+        clientFunc: async () => {
+            const moderations = await Client4.getChannelModerations(channelId);
+            return {channelId, moderations};
+        },
+        onSuccess: ChannelTypes.RECEIVED_CHANNEL_MODERATIONS,
+        params: [
+            channelId,
+        ],
+    });
+}
+
+export function patchChannelModerations(channelId: string, patch: Array<ChannelModerationPatch>): ActionFunc {
+    return bindClientFunc({
+        clientFunc: async () => {
+            const moderations = await Client4.patchChannelModerations(channelId, patch);
+            return {channelId, moderations};
+        },
+        onSuccess: ChannelTypes.RECEIVED_CHANNEL_MODERATIONS,
```

Comment:
should this have a failure action too?

Reply:
It only needs an `onFailure` if we specifically need to dispatch an action for that. If the caller is just waiting for the result of `patchChannelModerations`, then it shouldn't be needed.